lib/common/models/redis_store.rb

```diff
@@ -81,13 +84,16 @@ def self.keys
     end
 
     def self.delete(redis_key = nil)
+      secondary_redis.del(redis_key)
       redis_namespace.del(redis_key)
```

Comment:
This, and all of these, are really a transaction right? If one gets updated and one doesn't you're now out of sync.

And not that its _likely_ but if one succeeds and one fails, the method fails but maybe some data has been written.

The slippery slope here is you end up writing database-style WAL or use locking, both of which are not for the faint of heart.

Reply:
Has a `set` or `del` _ever_ failed so far to date that you know of? If it had, I would expect to see an error somewhere in sentry or maybe logs.